CryptoAnalysis/src/main/java/crypto/reporting/ErrorMarkerListener.java

```diff
@@ -54,24 +55,47 @@
 	protected final Table<SootClass, SootMethod, Set<AbstractError>> errorMarkers = HashBasedTable.create(); 
 	protected final Map<Class, Integer> errorMarkerCount = new HashMap<Class, Integer>();
 	protected final List<IAnalysisSeed> secureObjects = new ArrayList<IAnalysisSeed>();
+	/**
+	 * List for storing the package names to be ignored
+	 */
+	protected List<String> ignorePackagesList =  Collections.<String>emptyList();
 	
 	private void addMarker(AbstractError error) {
 		SootMethod method = error.getErrorLocation().getMethod();
 		SootClass sootClass = method.getDeclaringClass();
-		
 		Set<AbstractError> set = errorMarkers.get(sootClass, method);
+		Integer errorCount = errorMarkerCount.get(error.getClass());
```

Comment:
don't use `Interger` but `int`

Reply:
Changed type of errorCount from Integer to int.